src/main/kotlin/net/atos/zac/app/zaken/model/RESTZaakBetrokkeneGegevens.kt

```diff
@@ -17,8 +17,7 @@ data class RESTZaakBetrokkeneGegevens(
 
     var roltypeUUID: UUID,
 
-    @field:NotBlank
-    var roltoelichting: String,
+    var roltoelichting: String?,
```

Comment:
Does this not open the possibility that the `roltoelichting` will be provided but be an empty string? I.e. `roltoelichting=""`. That's something we would like to avoid if possible. Not sure OpenZaak will even accept that (it should not really). Maybe a bit of a stretch but still. I think for this we would need to write our own validation annotation. See e.g. https://stackoverflow.com/questions/31132477/java-annotation-for-null-but-neither-empty-nor-blank (but then in Kotlin of course). Would be useful here if feasible. Also ok to leave this for later I guess.

Reply:
This I think has been discussed by Felix and Karin. OpenZaak has 'toelichting' required. So at the actual call to that API the toelichting is escaped with a const string. Not sure if this answers your question btw.

Reply:
This could be an alternative: `toelichting = gegevens.roltoelichting?.ifEmpty(ROL_TOEVOEGEN_REDEN) ?: ROL_TOEVOEGEN_REDEN`

Reply:
>Not sure if this answers your question btw.

No. ;-)

Reply:
Implemented: `toelichting = gegevens.roltoelichting?.ifEmpty(ROL_TOEVOEGEN_REDEN) ?: ROL_TOEVOEGEN_REDEN`

Reply:
Applied @felixcicatt proposal

Reply:
OpenZaak does *not* accept empty string:
```
{'roltoelichting': [ErrorDetail(string='Dit veld mag niet leeg zijn.', code='blank')]}
```